packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -34,11 +34,17 @@ const selectors = {
 
 	// Plugin view
 	installButton: 'button:text("Install and activate")',
+	purchaseButton: 'button:text("Purchase and activate")',
 	deactivateButton: 'button:text("Deactivate")',
 	activateButton: 'button:text("Activate")',
 	openRemoveMenuButton: '.plugin-details-cta__manage-plugin-menu button[title="Toggle menu"]',
 	removeButton: '.popover__menu button:has-text("Remove")',
 
+	// Elegibility warnings
+	eligibilityWarning:
+		'.eligibility-warnings__title:has-text("Upgrade your plan to install plugins")',
+	eligibilityWarningContinueButton: 'button:text("Upgrade and activate plugin")',
+
```

Comment:
**comment:non-blocking** - Playwright has implemented new locator methods that eliminate the need  to keep these "selector objects" in the POMs. 

Methods like [`getByText`](https://playwright.dev/docs/locators#locate-by-text) can stand in for the selectors and make the class overall easier to follow and read.

Reply:
Thanks, @worldomonation, I will use the new methods from now on!